### PR TITLE
Update to beta5

### PR DESCRIPTION
**Pull request by hnrkndrssn**
Remove EF and Identity

### CODE REVIEW

File: src/JabbR/project.json [hunk 1]
```diff
@@ -1,40 +1,44 @@
...
+    "Microsoft.AspNet.Diagnostics": "1.0.0-beta4",
+    "Microsoft.AspNet.Server.IIS": "1.0.0-beta4",
+    "Microsoft.AspNet.Server.WebListener": "1.0.0-beta4",
+    "Microsoft.AspNet.SignalR.Server": "3.0.0-beta4",
+    "Microsoft.AspNet.StaticFiles": "1.0.0-beta4",
+    "Microsoft.Framework.ConfigurationModel.Json": "1.0.0-beta4",
+    "Microsoft.Framework.Logging": "1.0.0-beta4",
+    "Microsoft.Framework.Logging.Console": "1.0.0-beta4",
+    "Kestrel": "1.0.0-beta4",
+    "Microsoft.CodeAnalysis.CSharp": "1.0.0-beta4"
```
Comment (davidfowl) [inline src/JabbR/project.json]: Get rid of this

File: src/JabbR/project.json [hunk 2]
```diff
@@ -1,40 +1,44 @@
...
+  "frameworks": {
+    "dnx451": { },
+    "dnxcore50": { }
+  },
+  "exclude": [
+    "wwwroot",
+    "node_modules",
+    "bower_components"
+  ],
+  "packExclude": [
```
Comment (davidfowl) [inline src/JabbR/project.json]: publishExclude

File: src/JabbR/Startup.cs [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 using Microsoft.AspNet.Builder;
 using Microsoft.AspNet.Diagnostics;
 using Microsoft.AspNet.Hosting;
-using Microsoft.AspNet.Identity;
 using Microsoft.AspNet.Routing;
 using Microsoft.Framework.ConfigurationModel;
 using Microsoft.Framework.DependencyInjection;
 using Microsoft.Framework.Logging;
 using Microsoft.Framework.Logging.Console;
+using Microsoft.Framework.Runtime;
```
Comment (davidfowl) [inline src/JabbR/Startup.cs:10]: Why is this needed?

Comment (hnrkndrssn) [inline src/JabbR/Startup.cs:10]: Because `IApplicationEnvironment` :)
